### PR TITLE
Add release notes for v0.1.18

### PR DESCRIPTION
**Pull request by Gedochao**
#1584

### CODE REVIEW

**Review by tgodzik — APPROVED** [review]
LGTM!

**Comment by Gedochao** [comment]
Rebased & added in references to #1617 & #1531

**Comment by Gedochao** [comment]
Rebased & added in references to #1604

File: website/docs/release_notes.md [hunk 1]
```diff
@@ -5,6 +5,167 @@ sidebar_position: 99
 
 # Release notes
 
+## [v0.1.18](https://github.com/VirtusLab/scala-cli/releases/tag/v0.1.18)
+
+## Filter tests with `--test-only`
+It is now possible to filter tests with the `--test-only` option.
```
Comment (lwronski) [inline website/docs/release_notes.md]: Add more precisely, that `--test-only` filter only the tests suite. If someone wants to filter the name of tests to run, he has to pass arguments to framework args, for example in `munit`:
```
-- '*foo*'
```

Comment (Gedochao) [reply website/docs/release_notes.md]: should be fixed now

**Pull request merged**